tests/acceptance/08_commands/01_modules/module_with_no_output.cf.sub

```diff
@@ -0,0 +1,33 @@
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+}
+
+bundle agent test
+{
+  commands:
+      "$(G.cat) $(G.testdir)/module.output.txt"
+        module => "true",
+        contain => test_contain("false");
+
+      "$(G.cat) $(G.testdir)/module.no_output.txt"
+        module => "true",
+        contain => test_contain("true");
+
+      "$(G.cat) $(G.testdir)/module.default.txt"
+        module => "true";
+
+      "$(G.cat) $(G.testdir)/no_module.output.txt"
+        contain => test_contain("false");
+
+      "$(G.cat) $(G.testdir)/no_module.no_output.txt"
```

Comment:
They are generated in the non-`.sub` file. I didn't want to clutter the directory with all these small oneline files.